lib/watir-screenshot-stitch.rb

```diff
@@ -78,7 +73,7 @@ def one_shot?
       end
 
       def bug_shot?
-        return false unless @browser&.name == :firefox
```

Comment:
I'm curious what the motivation for removing the safety is here?

Reply:
I just pushed a commit to change this back and set minimum Ruby to 2.3. Watir hasn't enforced that, yet, but we'll be doing it soon.